VTEX - Logistics API.json

```diff
@@ -6996,279 +6988,166 @@
-                                                    "neighborhood": {
-                                                        "type": "string",
-                                                        "description": "Neighborhood."
-                                                    },
-                                                    "street": {
-                                                        "type": "string",
-                                                        "description": "Street name."
-                                                    },
-                                                    "number": {
-                                                        "type": "string",
-                                                        "description": "Pickup point address number."
-                                                    },
-                                                    "complement": {
-                                                        "type": "string",
-                                                        "description": "Address complement."
-                                                    },
-                                                    "location": {
-                                                        "type": "object",
-                                                        "description": "Geolocation properties.",
-                                                        "properties": {
-                                                            "latitude": {
-                                                                "type": "number",
-                                                                "description": "Latitude."
-                                                            },
-                                                            "longitude": {
-                                                                "type": "number",
-                                                                "description": "Longitude."
-                                                            }
-                                                        },
-                                                        "example": {
-                                                            "latitude": -22.974477767944336,
-                                                            "longitude": -43.18672561645508
-                                                        }
-                                                    }
-                                                }
-                                            },
-                                            "isActive": {
-                                                "type": "boolean",
-                                                "description": "Indicates whether pickup point is active."
+                                        "$ref": "#/components/schemas/ListPickupPointById"
```

Comment:
Nice choice to use the schema here 😄 It might be good to rename it later on to something like PickupPoint (if available) so that it better represents more than one endpoint where it's used instead of just ListPickupPointById